### PR TITLE
AyyyKayyy 1S whoop race tune

### PR DESCRIPTION
**Pull request by limonspb**
1S whoop brushless tune for acro racing.

AyyyKayyy - Whooptopia winner 2019, 2020, and 2021. Whooptopia - biggest and the most intense whoop race in the USA.

This tune does not require RPM and works with regular BLHELI_S

### CODE REVIEW

File: presets/4.3/tune/whoop_ayyykayyy.txt [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+set iterm_relax_cutoff = 20
+
+# -- iTerm rotation
+set iterm_rotation = ON
+
+# -- TPA (default)
+set tpa_breakpoint = 1250
+
+# -- Feedforward
+set feedforward_transition = 21
```
Comment (sugaarK) [inline presets/4.3/tune/whoop_ayyykayyy.txt:64]: We don’t recommenced using ff trans in 4.3 it’s been superseded by jitter_reduction

**Review by sugaarK — COMMENTED** [review]
Not sure about using ff transition jitter reduction is the preferred  way to do this now

File: presets/4.3/tune/whoop_ayyykayyy.txt (hunk 1 above)
Comment (limonspb) [reply presets/4.3/tune/whoop_ayyykayyy.txt:64]: > Not sure about using ff transition jitter reduction is the preferred way to do this now

Yeah agree. I will ask. But if the author says that's intentional, we should let it be I think.

Comment (limonspb) [reply presets/4.3/tune/whoop_ayyykayyy.txt]: @sugaarK  good catch.
Removed

**Comment by limonspb** [comment]
> Not sure about using ff transition jitter reduction is the preferred way to do this now

good catch.
Removed

File: presets/4.3/tune/whoop_ayyykayyy.txt [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+set iterm_relax_cutoff = 20
+
+# -- iTerm rotation
+set iterm_rotation = ON
+
+# -- TPA (default)
+set tpa_breakpoint = 1250
+
+# -- Antigravity
+set anti_gravity_gain = 0
```
Comment (sugaarK) [inline presets/4.3/tune/whoop_ayyykayyy.txt:64]: You sure you want  antigravity set to zero in 4.3, It works different to 4.2

**Review by sugaarK — CHANGES_REQUESTED** [review]

File: presets/4.3/tune/whoop_ayyykayyy.txt (hunk 2 above)
Comment (limonspb) [reply presets/4.3/tune/whoop_ayyykayyy.txt:64]: > You sure you want antigravity set to zero in 4.3, It works different to 4.2

let me ask lol

Comment (limonspb) [reply presets/4.3/tune/whoop_ayyykayyy.txt:64]: > You sure you want antigravity set to zero in 4.3, It works different to 4.2

@sugaarK  i just flown this tune, its a real fire. it does not need AG :)

**Review by sugaarK — APPROVED** [review]

**Pull request merged**